Update survey list styles

dan-nyanko opened the pull request:
the images (.png) used for checkbox on/off will need to be created to match the mockups

dan-nyanko committed:
made some changes to the mock data

dan-nyanko committed:
reactive title based on updating the props

dan-nyanko committed:
created a prop setTitle available to all views

dan-nyanko committed:
updating styles

dan-nyanko committed:
styled the list, added checkbox, set the view title

dan-nyanko committed:
Merge branch 'master' into update-survey-list-styles

sophiabersani commented:
Looking good and functional so far.
- To match the design specs for those checkboxes: this checkbox package you found lets us use the pseudo-css for styling. Perhaps implementing it with an open-source icon package such as https://github.com/oblador/react-native-vector-icons would help us avoid ugly pixelated PNG problems?
- I'd suggest creating a new component for the Survey list items to facilitate the left-and-right sliding our design docs ask for. Putting everything inside the view might become cumbersome as code grows.

Thoughts?

dan-nyanko commented:
:thumbsup:

dan-nyanko committed:
nightly, converting checkbox 3rd party package to use components

- added react-native-vector-icons
- copied ttf fonts to android
- update android build.gradle version
- temporary js/lib/checkbox.js and check.png
- update SurveyListPage.js to use custom checkbox fork

dan-nyanko committed:
removing a bunch of unused fonts from android

dan-nyanko committed:
adding FontAwesome to iOS

dan-nyanko committed:
updated by AndroidStudio, may need to .gitignore

dan-nyanko committed:
removing temporary development files

dan-nyanko committed:
adding the faded colors from the screen mockups

dan-nyanko committed:
pass props for the checked and unchecked components

dan-nyanko committed:
make the survey list item a component

dan-nyanko committed:
Merge branch 'master' into update-survey-list-styles
